src/work-allocation/containers/available-tasks/available-tasks.component.ts

```diff
@@ -79,10 +79,9 @@ export class AvailableTasksComponent extends TaskListWrapperComponent {
    */
   public claimTaskAndGo(task: Task): void {
     this.taskService.claimTask(task.id).subscribe(() => {
-      // constant below removes spaces from caseReference to get caseId
-      const caseId = task.caseReference.replace(/\s/g, '');
+      const goToCaseUrl = `/cases/case-details/${task.caseReference}`
```

Comment:
This looks risky. I thought the case reference was a _formatted_ version of the case ID...?

Reply:
Currently the data being passed through isn't formatted at all - we get a 16 digit number with no spaces from API which is currently being used. If we want to format it I can add some functionality to add spaces (similar to the formatting we had before with the data 'XXXX XXXX XXXX XXXX') and use the replace again. We don't actually get a 'caseReference' from the API, we only get a 'case_id' which has no formatting.

Reply:
Hmmm... then I think we should continue to refer to it as a `caseId` (or `case_id`) rather than `caseReference` as that's what it is. 🤷‍♂️